apps/portal/src/app/shared/services/file.service.ts

```diff
@@ -0,0 +1,42 @@
+import { Injectable } from '@angular/core';
```

Comment:
This should not be a shared service.
Create a service file along with the component no need for separate folder. its related to analyzer feature.

Reply:
I will be using this service for uploading file too, that's why I kept it here because I will be using this service in both component, so do I create a two separate services for uploading file and fetching status? or keep this in shared considering I will be using this for other functionality too

Reply:
Okay for now.

But we basically just have one page/module.
The main app page with two child components used for it, the upload component and the list component.
So ideally we just use summary service(because it is related to summary entity and APIs are called for it to not file)
So ideal would me in summary component, you will have  sub folders for components create-summary, list-summary, and have a service file.
```
summary component(parent main page)
---summary.service
---create-summary/component files
---list-summary/component-files 
```

Reply:
Okay, for now, can we merge the PR if this is the only issue remaining, so I can finish the upload API integration? We can change the file structures after that?